layers/+completion/auto-completion/packages.el

```diff
@@ -208,14 +206,16 @@
         (push spacemacs-layer-snippets-dir yas-snippet-dirs)
         ;; ~/.emacs.d/elpa/yasnippet-xxxxx/snippets
         (push 'yas-installed-snippets-dir yas-snippet-dirs)
+        ;; ~/.emacs.d/private/snippets
+        (push emacs-directory-snippets-dir yas-snippet-dirs)
         ;; ~/.spacemacs.d/snippets
         (when dotspacemacs-directory-snippets-dir
           (push dotspacemacs-directory-snippets-dir yas-snippet-dirs))
         ;; arbitrary directories in `auto-completion-private-snippets-directory'
-        (when private-yas-dir
-          (if (listp private-yas-dir)
-              (setq yas-snippet-dirs (append yas-snippet-dirs private-yas-dir))
-            (push private-yas-dir yas-snippet-dirs))))
+        (when auto-completion-private-snippets-directory
+          (if (listp auto-completion-private-snippets-directory)
+              (setq yas-snippet-dirs (append yas-snippet-dirs auto-completion-private-snippets-directory))
+            (push auto-completion-private-snippets-directory yas-snippet-dirs))))
```

Comment:
We should switch `push` functions to `add-to-list` because I'm afraid that any config reloading with `SPC f e R` may append the same directories again and over again.

This remark is for all the `push` in this `use-package` sexp.